tests/test_cli.py

```diff
@@ -168,3 +169,46 @@ def test_check_datapack_not_complies():
     result = runner.invoke(cli, ["check-datapack", str(datapack)])
     assert result.exit_code == exit_codes.DATAPACK_SPEC_ERROR != 0
     assert "DataPackSpecError" in result.stderr
+
+
+@pytest.mark.parametrize(
+    "props,example_suffix", [([], "_wo_props.mm.txt"), (["-p"], "_w_props.mm.txt")]
```

Comment:
Also check with the long arguments i.e. here `--with-properties` to ensure that the interface is stable. Might also be worth to align with the solutions chosen above to make the test look more uniform.

Reply:
Fixed in 5f3daaf90ba97fcf6840536e7be9f17c6483f83d